src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import App from "./App";
```

Comment:
The UI should not jump

https://user-images.githubusercontent.com/34886027/208317326-96d11ed6-5682-43f1-9d3e-02923f9c2ce0.mov